src/main/java/wolox/training/controllers/BookController.java

```diff
@@ -0,0 +1,17 @@
+package wolox.training.controllers;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+
+@Controller
```

Comment:
I recommend to use RestController annotation, it's newer and it includes ResponseBody and Controller behavior

Reply:
Controller annotation BookController was updated to RestController